docs/getting-started-guides/docker-multinode.md

```diff
@@ -74,10 +74,11 @@ it is still useful to use containers for deployment and management, so we create
 You can specify the version on every node before install:
 
 ```sh
-export K8S_VERSION=<your_k8s_version (e.g. 1.2.0-alpha.6)>
+export K8S_VERSION=<your_k8s_version (e.g. 1.2.0-alpha.7)>
```

Comment:
In the future, the values in the docs are going to be munged in some way.
But in this script, I don't know. `curl -sSL https://console.cloud.google.com/storage/kubernetes-release/release/latest.txt` returns a login form, so can't be used right now

This is just an update so `docker-multinode` works, since #17213 is merged.